.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: false # enable container-based infrastructure
+
+addons:
+  apt:
+    sources:
+      - deadsnakes # source required so it finds the package definition below
+    packages:
+      # packahe list:
+      # https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise
+      - python3.5 python3.5-dev python3.5-venv check libglew-dev
```

Comment:
but without it, `bootstrap` complains. Maybe there's an option to not do dependencies test. But i dont see why we shouldn't do them. As long as deps don't have their own continuous integration tests, we should at least do them here